### ISSUE TITLE
Problem compiling with latest node-sass when using multiple rules

### ISSUE THREAD

**Issue by joseluis** [eduardoboucas/include-media#37]
Compilation doesn't finish and all RAM is consumed in the process, when using multiple rules (that includes using $media-expressions)

minimal example.scss:
```
@import 'bower_components/include-media/dist/include-media';

@include media( ">20rem", "<40rem" ) { 
    body{
        color: blue;
    }   
}
```

I've tested it and the problem happens with include-media versions 1.2, 1.1.2 and 1.0.0.

In Ubuntu 14.04, using node.js 10.38 and these versions of node-sass (libsass):
- node-sass 3.1.1 (libsass 3.2.4)
- node-sass 3.0.0 (libsass 3.2.2),

But it compiles succesfully with:
- node-sass 2.1.1 (libsass 3.1.0)

Steps to reproduce:
```
npm install node-sass@3.1.1
bower install include-media#1.1.2

./node_modules/.bin/node-sass example.scss
```
(To see the available versions: `npm info node-sass`)

I'm sending the bugreport here, but upon confirmation I believe it'd be better if you be the one who sends the report to either library, or both.

**EDIT** I've realized this happens when using multiple arguments, not justs media-expressions

**Comment by HugoGiraudel** [comment]
Seconded.

**Comment by eduardoboucas** [comment]
Fixed in #39. Thanks @HugoGiraudel!
